packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -214,8 +215,8 @@ export class ElectronMainApplication {
      *
      * @param options
      */
-    async createWindow(asyncOptions: MaybePromise<TheiaBrowserWindowOptions> = this.getDefaultBrowserWindowOptions()): Promise<BrowserWindow> {
-        const options = await asyncOptions;
+    async createWindow(options: TheiaBrowserWindowOptions = this.getDefaultTheiaWindowOptions()): Promise<BrowserWindow> {
```

Comment:
Why did you change the type from `MaybePromise<TheiaBrowserWindowOptions>` to `TheiaBrowserWindowOptions`?

Reply:
I told to myself that the getDefaultBrowserWindowOptions() wasn't a promise. It returns only default options without waiting for them.

Reply:
Please revert the type and keep the original method name. Thank you!

Reply:
@kittaakos I agree that this should be reverted in the context of that PR, but long term I think the method should just take the `TheiaBrowserWindowOptions` and not a promise. If someone deals with a promise, they can resolve it first before calling `createWindow`. Is there anything preventing us from doing that, besides being breaking?